index.html

```diff
@@ -79,16 +79,15 @@ <h3>Graphic Design</h3>
           </ul>
         </div>
       </section>
+
       <!-- Right Mid -- Three divs with several smallers sections inside the mid one, and two in the lower bottom -->
-      <section class="mid_right parent">
+      <section class="mid_right">
 
-        <div class="child">
-          <h2>Work History</h2>
-        </div>
+        <h2>Work History</h2>
 
-        <div class="child">
-          <table>
-            <tr>
+        <div class="table">
+          <table id="table">
```

Comment:
nice differentiation haha ones a class ones a id. I would suggest using different names in the future